index.bs

```diff
@@ -10316,18 +10318,14 @@ an exception when called as a function.
         [=overload resolution algorithm=].
     1.  Let |R| be the result of performing the actions listed in the description of
         |constructor| with |values| as the argument values.
-    1.  Return the result of [=converted to an ECMAScript value|converting=]
+    1.  Rethrow any [=exception=] thrown in the previous step.
```

Comment:
> Not sure if a constructor ever does that

It doesn't even have to be the constructor itself.  For example, step 4 (invoking the overload resolution algorithm) can trigger arbitrary page-provided code which can throw whatever the page wants.  In general, so can step 5, if the constructor takes an `object` argument and pokes at it in any way.